xwiki-commons-core/xwiki-commons-job/xwiki-commons-job-default/src/test/resources/jobs/status/id1/id2/id3/&status/status.xml

```diff
@@ -19,7 +19,7 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<org.xwiki.job.internal.DefaultJobStatus>
+<org.xwiki.job.DefaultJobStatus>
```

Comment:
I had to modify the serialization in the XML files of the test, since the class is not present anymore in this module. I don't think it would have an impact for real world status serialization if the old class is available through the legacy module.